.env.example

```diff
@@ -0,0 +1 @@
+PYTHONPATH=/<absolute_fs_path>/mpyl/src
```

Comment:
Should we also add any of these? Some might be outdated, you probably know better than me

```
JENKINS_URL=""
JIRA_URL=""
DOCKER_HOST_NAME=""
CHANGE_ID="1"
GITHUB_TOKEN=""
GITHUB_JOB=""
SLACK_TOKEN=""
MPYL_GITHUB_APP_PRIVATE_KEY=""
MPYL_VERSION=""
```

Reply:
Those other env vars relate to mpyl run configuration. They could also be injected by Jenkins or another task runner execution time. You will also get clear(er) feedback when config values are wrong or missing.
So I don't think there's much point to listing them here as well as in `mpyl_config`